h2o-core/src/main/java/hex/Model.java

```diff
@@ -1625,9 +1644,30 @@ public Frame score(Frame fr, String destination_key, Job j, boolean computeMetri
           CategoricalWrappedVec.updateDomain(output.vec(0), sdomain);
       }
     }
-    Frame.deleteTempFrameAndItsNonSharedVecs(adaptFr, fr);
+    for (Frame tmp : tmpFrames) Frame.deleteTempFrameAndItsNonSharedVecs(tmp, fr);
     return output;
   }
+  
+  private void applyPreprocessors(Frame fr, List<Frame> tmpFrames) {
+    if (_parms._preprocessors == null) return;
+    
+    for (Key<ModelPreprocessor> key : _parms._preprocessors) {
+      DKV.prefetch(key);
```

Comment:
what is the benefit of this call?

Reply:
It's a technic I learnt from Michal when you need to load multiple keys:
- one loop to prefetch
- one loop to use the fetched objects: this way when you're using the first one, the others can be deserialized in the background, so this loop is faster then.